roles/pulp_webserver/tasks/apache.yml

```diff
@@ -6,6 +6,21 @@
         name: "{{ pulp_webserver_apache_package }}"
         state: present
 
+    - debug:
+        msg: Pulp api bind is now (before change) {{ pulp_api_bind }}
+
+    - name: Change unix socket to ip
```

Comment:
I think, you can male apache use the socket too.

https://httpd.apache.org/docs/trunk/mod/mod_proxy.html#proxypass